### DISCUSSION TITLE
Fuel flow monitoring hardware

### DISCUSSION THREAD

**Discussion by big-col** [hatlabs/discussions#95] [General]
Hey guys, what fuel flow hardware are you using? We have rather good but very old FloScan system currently, but this won't last forever and one of the LCD screens is already showing signs of failure.  

I have a couple of the new Halmet hats, but need to understand what online fuel flow monitors will work with this please.

Thanks
Colin.

**Comment by kristianschneider** [comment]
Oh. I'm very curious on this too. Did you end up finding something suitable ?
